client/coral-admin/src/actions/auth.js

```diff
@@ -10,13 +10,20 @@ const checkLoginFailure = error => ({type: actions.CHECK_LOGIN_FAILURE, error});
 export const checkLogin = () => dispatch => {
   dispatch(checkLoginRequest());
   coralApi('/auth')
-    .then(user => {
-      const isAdmin = !!user.roles.filter(i => i === 'admin').length;
-      dispatch(checkLoginSuccess(user, isAdmin));
+    .then(result => {
+      if (result.csrfToken !== null) {
+        dispatch(check_csrf(result.csrfToken));
+      }
+
+      const isAdmin = !!result.user.roles.filter(i => i === 'admin').length;
```

Comment:
same as

```
const isAdmin = result.user.roles.some(i => i === 'admin')
```

or

```
const isAdmin = result.user.roles.indexOf('admin') !== -1
```

I'd suggest to add this as a separate helper that could be consumed from different places